pages/Documents/Reporting/NetHandleTimeAPI/Methods/get-net-handle-time-by-skill.md

```diff
@@ -15,13 +15,13 @@ Some metrics are calculated only in 60-minute intervals.
 These metrics, when calling the API in a 15-minutes interval, will be replicated for each timeframe.
 
 {: .note}
-In case there is no data available, the response element will contain -1.
+In case there is no data available, the response element will contain 0 (On September 1st 2022, the  default values were changed from -1 to 0)..
```

Comment:
The periods are in the wrong places. It seems what’s needed is something like this:

`In case there is no data available, the response element will contain 0. (On September 1st 2022, the default values were changed from -1 to 0.)`